pkg/kor/configmaps.go

```diff
@@ -126,7 +156,13 @@ func processNamespaceCM(clientset kubernetes.Interface, namespace string, filter
 	}
 
 	var usedConfigMaps []string
-	slicesToAppend := [][]string{volumesCM, envCM, envFromCM, envFromContainerCM, envFromInitContainerCM}
+	slicesToAppend := [][]string{
+		volumesCM,
+		envCM,
+		envFromCM,
+		envFromContainerCM,
+		envFromInitContainerCM,
+	}
```

Comment:
Autoformat by IDE

Reply:
Yeah. We need to enforce it in the CI.
I'll open a task for it

Reply:
Strange. Intellij doesn't reformat to this.

Reply:
There might be some differences in how IntelliJ and GoLand work when it comes to formatting. I’m using the default code style configuration, no custom tweaks there.